circle.yml

```diff
@@ -1005,8 +1005,7 @@ jobs:
           command: node cli/bin/cypress info --dev
       - store-npm-logs
 
-  # a special job that keeps polling Circle and when all
-  # individual jobs are finished, it closes the Percy build
+  # closes the Percy build when required jobs are passing
   percy-finalize:
```

Comment:
There are more updates here than built steps that must finalize before the `percy-finalize` step is ran.  I think we need to add `npm-design-system` as a [required job](https://github.com/cypress-io/cypress/blob/54faadcf2b17b310d36f600276a991f53ff87f04/circle.yml#L2056) before running percy-finalize`.

Reply:
Oh let's add that for sure 👍  will do now

Can you clarify your first sentence? The change to the comment was mainly because, since we aren't polling at all (and haven't been for a while), the comment is stale and was a bit misleading when I started looking into this. I'm all for describing better what this is doing though, if you want to suggest a different comment here.

Reply:
My bad, it was just the one update. I thought there was two missing originally and should have re-read that.